src/image_version.py

```diff
@@ -0,0 +1,27 @@
+class ImageVersion:
+    def __init__(self, image_full_version: str, image_type: str):
```

Comment:
can we accomplish this functionality using the [semver Version class](https://github.com/aws/sagemaker-distribution/blob/main/src/main.py#L13) ? I just think it might be a little confusing to be using both in different places in the package

Reply:
+1 to this point

Reply:
good point, updated